eslint/backend.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    extends: [
+        "./index.js",
+        "plugin:@typescript-eslint/recommended",
+        "prettier/@typescript-eslint"
+    ],
+    plugins: [
+        "typescript-eslint",
+    ],
+    parser: "@typescript-eslint/parser",
```

Comment:
Had to use these option to have the parser working correctly. Not sure  it's still necessary:
`"parserOptions": {
        "ecmaVersion": 2018,
        "sourceType": "module"
    },`